legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-bigquery/legend-engine-xt-relationalStore-bigquery-wrapper/src/main/java/com/deloitte/legend/engine/bigquery/BigQueryConnection.java

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright 2020 Google LLC
+ */
+
+package com.deloitte.legend.engine.bigquery;
```

Comment:
Should this be com.google ? Also, this class is technically not a part of the "engine". 

Can we rename this to com.google.bigquery.jdbc